public/src/app.js

```diff
@@ -276,7 +276,11 @@ app.cacheBuster = null;
 			if (err) {
 				return app.alertError(err.message);
 			}
-			app.openChat(roomId);
+			if (!ajaxify.currentPage.match(/^chats\//)) {
```

Comment:
@barisusakli this need to be changed to `startsWith('chats')`

It doesn't work when you go to the expanded chat do not have a room picked as ajaxify.currentPage only returns `chats`